driver/src/price_finding/price_finder_interface.rs

```diff
@@ -36,16 +37,50 @@ impl FromStr for SolverType {
             "standard-solver" => Ok(SolverType::StandardSolver),
             "fallback-solver" => Ok(SolverType::FallbackSolver),
             "naive-solver" => Ok(SolverType::NaiveSolver),
+            "open-solver" => Ok(SolverType::OpenSolver),
             _ => Err(anyhow!("solver type does not exit")),
         }
     }
 }
 
 impl SolverType {
-    pub fn to_args(self) -> Vec<String> {
+    pub fn to_args(self, result_folder: &str, input_file: &str, time_limit: String) -> Vec<String> {
+        let mut standard_solver_args: Vec<String> = vec![
+            String::from("-m"),
+            String::from("scripts.e2e._run"),
+            format!("{}{}", "/app/", input_file.to_owned()),
+            format!("--outputDir={}{}", "/app/", result_folder),
+            format!("--solverTimeLimit={}", time_limit),
+        ];
+        let open_solver_args = vec![
+            String::from("-m"),
+            String::from("src.match"),
+            format!("{}{}", "/app/", input_file.to_owned()),
+            format!(
+                "--solution={}{}{}",
+                "/app/",
+                result_folder.to_owned(),
+                "06_solution_int_valid.json",
+            ),
+            String::from("best-token-pair"),
+        ];
         match self {
-            SolverType::StandardSolver => vec![],
-            SolverType::FallbackSolver => vec![String::from("--useExternalPrices")],
+            SolverType::OpenSolver => open_solver_args,
+            SolverType::StandardSolver => standard_solver_args,
```

Comment:
I would propose that there is no reason to declare these args variables and that they could go directly within the match statement (since they don't appear to be used anywhere else).

Reply:
I want to use the variable `standard_solver_args`. I use it in the next line

Reply:
Ok, if it is used twice, but what about the `open_solver_args`?